src/java/org/apache/cassandra/index/sai/plan/TopKProcessor.java

```diff
@@ -89,6 +90,8 @@
  */
 public class TopKProcessor
 {
+    public static final String INDEX_MAY_HAVE_BEEN_DROPPED = "An index may have been dropped. Ordering on non-clustering " +
+                                                             "column requires the column to be indexed";
```

Comment:
Super nit: add a period at the end of the second phrase
```suggestion
                                                             "column requires the column to be indexed.";
```